templates/components/common/payment-icons.html

```diff
@@ -1,30 +1,70 @@
 {{#with theme_settings}}
     {{#or show_accept_amex show_accept_discover show_accept_mastercard show_accept_paypal show_accept_visa show_accept_amazonpay show_accept_googlepay show_accept_klarna}}
-    <div class="footer-payment-icons">
+    <ul class="footer-payment-icons" aria-label="{{lang 'account.payment_methods.payment_icons_label'}}">
         {{#if show_accept_amex}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-american-express"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>American Express</title>
+                <use href="#icon-logo-american-express"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_discover}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-discover"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Discover</title>
+                <use href="#icon-logo-discover"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_mastercard}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-mastercard"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Mastercard</title>
+                <use href="#icon-logo-mastercard"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_paypal}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-paypal"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Paypal</title>
+                <use href="#icon-logo-paypal"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_visa}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-visa"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Visa</title>
+                <use href="#icon-logo-visa"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_amazonpay}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-amazonpay"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Amazonpay</title>
+                <use href="#icon-logo-amazonpay"></use>
+            </svg>
+        </li>
         {{/if}}
         {{#if show_accept_googlepay}}
-        <svg class="footer-payment-icon"><use href="#icon-logo-googlepay"></use></svg>
+        <li>
+            <svg class="footer-payment-icon" role="img">
+                <title>Googlepay</title>
```

Comment:
I believe better change it to `Google Pay`

Reply:
Done!

Reply:
@VladDlogush can you please update `Amazon Pay` also

Reply:
Now is better!